Source/WebCore/rendering/Grid.h

```diff
@@ -123,6 +126,8 @@ class GridIterator {
     static GridIterator createForSubgrid(const RenderGrid& subgrid, const GridIterator& outer);
 
     RenderBox* nextGridItem();
+    Vector<RenderBox*> masonryItems() { return m_grid.autoMajorAxisAutoGridItems; };
```

Comment:
Does this need to copy the Vector? It should be `const`

Reply:
Going to remove this and re-perform the computation in `resolveIntrinsicTrackSizes`.